### ISSUE TITLE
Browsing security doesn't work

### ISSUE THREAD

**Issue by vozersky** [AdguardTeam/AdguardBrowserExtension#580]
1. Enable safebrowsing.
2. Open any page from safebrowsing filter, for example: http://www.internetbadguys.com/
Result: Adguard doesn't show a Browsing security warning.

Also, got this in console log once:

```
2017-03-09T13:58:37.504Z: Error response status 502 received from safebrowsing lookup server.
log.js (71,9)
```

```
HTTP502: BAD GATEWAY - The server, while acting as a gateway or proxy, received an invalid response from the upstream server it accessed in attempting to fulfill the request.
(XHR)GET - https://sb.adtidy.org/safebrowsing-lookup-hash.html?prefixes=6C9B2024%2F3AA8C51F
```

**Comment by MarDarkXT** [comment]
It works for me.
![browsersecurity](https://cloud.githubusercontent.com/assets/25929539/23755594/8b0cbb44-04e9-11e7-986a-c85bfaecc28e.PNG)

**Comment by ameshkov** [comment]
@vozersky that's just @mmigi was re-deploying safebrowsing server.

**Comment by MarDarkXT** [comment]
@ameshkov I think he is right. It only worked the first time. Now every time I access the page it doesn't work anymore.

**Comment by vozersky** [comment]
@ameshkov I know he was. Still doesn't work for me though.

@MarDarkXT are you on v2.6.0?

**Comment by MarDarkXT** [comment]
@vozersky No on 2.5.18, but it still happens on this version too.

**Comment by vozersky** [comment]
@MarDarkXT well, if you choose "advanced" -> "proceed anyway" on some website, then your choice is stored in browser cache and you will not see the warning again until you clean the cache

**Comment by MarDarkXT** [comment]
@vozersky No I didn't. It is like this.
1. Open browser
2. Visit GitHub issue
3. Click on link
4. Tab opens and warning is shown
5. Close tab
6. Click on link again. It will always show the page until I close the browser.

**Comment by Mizzick** [comment]
Works fine for me.

**Comment by atropnikov** [comment]
In Edge browser, for updating tab's URL we use content script due to some restrictions, that are described here: https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/tabs/update#Parameters
The problem may appear when content script have not been loaded yet.

**Comment by atropnikov** [comment]
For Firefox browser, this task is resolved with #583
